apps/backend/src/lib/utils/constants.ts

```diff
@@ -1,7 +1,7 @@
 export const HOUR24 = 24 * 60 * 60 * 1000;
 const MOCK_API_BASE_URL = process.env?.MOCK_API_BASE_URL
 	? process.env.MOCK_API_BASE_URL
-	: "http://localhost:3005/api/";
+	: "http://localhost:3000/";
```

Comment:
Is this correct?

Reply:
This is just picked when the env variable is not specified. However, in our case, the env is specified. So there shouldn't be any issues.